pkg/toolsets/lvms/lvms_capacity.go

```
@@ -0,0 +1,361 @@
+	return result.String()
+}
+
+// parseCapacity parses a Kubernetes quantity string to bytes
+func parseCapacity(s string) int64 {
+	if s == "" {
+		return 0
+	}
+	// Simple parsing for common formats
+	var value int64
+	var unit string
+	fmt.Sscanf(s, "%d%s", &value, &unit)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Sscanf` is not checked

(errcheck)